test/nodes/test_webhook_scheduler.py

```
@@ -0,0 +1,1065 @@
+_mock_rocketride = MagicMock()
+_mock_rocketride.CONST_WS_PING_INTERVAL = 20
+_mock_rocketride.CONST_WS_PING_TIMEOUT = 20
+_inject('rocketride', _mock_rocketride)
+
+# rocketlib
+_mock_rocketlib = MagicMock()
+_inject('rocketlib', _mock_rocketlib)
+
+# depends -- must be a real module with a callable ``depends`` attribute
+_mock_depends_mod = ModuleType('depends')
+_mock_depends_mod.depends = lambda *a, **kw: None  # type: ignore[attr-defined]
```

Ruff (0.15.7)

[warning] Unused lambda argument: `a`

(ARG005)